Workbooks/Resource Insights/AzureDataExplorerCluster/ResourceInsights.workbook

```diff
@@ -149,8 +144,9 @@
                 "{Resource}"
               ],
               "timeContext": {
-                "durationMs": 43200000
+                "durationMs": 0
```

Comment:
just double checking if we wanted a time duration of 0? Or I guess this is replaced with TimeRange param